### PR TITLE
Added Assumptions to the report

### PR DESCRIPTION
**Pull request by SanyaDua**

### CODE REVIEW

File: Report.md [hunk 1]
```diff
@@ -31,6 +31,14 @@ Problem Definition
 ## **Assumptions**
  5. List assumptions you have made in the systems analysis.
```
Comment (zachartmann) [inline Report.md]: We're just removing these lines
```suggestion
```

File: Report.md [hunk 2]
```diff
@@ -31,6 +31,14 @@ Problem Definition
...
```
Comment (zachartmann) [inline Report.md:33]: Maybe write an introductory line like "Over the course of system analysis, there were a few assumptions we had to make in order to understand the system as a whole. These assumptions are listed below:"
[I was actually doing this at the same time so I just stole it ahaha]

**Review by zachartmann — APPROVED** [review]
Thanks for doing this, just a comment or two

File: Report.md (hunk 2 above)
Comment (SanyaDua) [reply Report.md:33]: Sure, I will add that in.  Thanks!

File: Report.md [hunk 3]
```diff
@@ -134,6 +142,8 @@ Workproducts and Models
 =======================
 6. Document your proposed workproducts and models *and descriptions*.
 
+The Collaboration Diagram for Customer emphasises the structure of interactions between the Customer, Relationship manager, Call Management Centre(CMC) and Interactive voice Response unit. The diagram defines the role of each object or actor.  This diagram  follows Customer’s journey from calling the Call Management Centre through to the payment confirmation text for the order.
```
Comment (seant1) [inline Report.md:145]: just added a space between "Centre" and "(CMC)"
```suggestion
The Collaboration Diagram for Customer emphasises the structure of interactions between the Customer, Relationship manager, Call Management Centre (CMC) and Interactive voice Response unit. The diagram defines the role of each object or actor.  This diagram  follows Customer’s journey from calling the Call Management Centre through to the payment confirmation text for the order.
```

**Review by seant1 — APPROVED** [review]
Great work 🎉

**Pull request merged**